CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 ## 5.23.0 (Unreleased)
 
+NOTES:
+* provider: introduced support for provider-defined functions. This feature is in Terraform v1.8.0+. ([#7153](https://github.com/hashicorp/terraform-provider-google-beta/pull/7153))
```

Comment:
```suggestion
* provider: introduced support for [provider-defined functions](https://developer.hashicorp.com/terraform/plugin/framework/functions). This feature is in Terraform v1.8.0+. ([#7153](https://github.com/hashicorp/terraform-provider-google-beta/pull/7153))
```
given it's completely new to the provider, I think a link to Hashicorp's documentation would be useful for users